database.json

```diff
@@ -1186,6 +1192,12 @@
     "repo": "dso",
     "desc": "dso is a simple ORM Library based on deno_mysql"
   },
+  "duckduckgo": {
+    "type": "github",
+    "owner": "0vw",
+    "repo": "duckduckgo",
+    "desc": "A Deno wrapper for the DuckDuckGo API"
```

Comment:
Unrelated changes. Please merge with master

Reply:
I have updated it according to the latest database.json
I am not sure what you mean by "Please merge with master", I assume that?
Let me know further